configure.ac

```diff
@@ -5,7 +5,7 @@
 ## Please see the COPYING and CONTRIBUTORS files for details.
 ##
 
-AC_INIT([Squid Web Proxy],[5.0.0-BZR],[http://bugs.squid-cache.org/],[squid])
+AC_INIT([Squid Web Proxy],[5.0.0-VCS],[http://bugs.squid-cache.org/],[squid])
```

Comment:
The snapshots from master branch are alpha releases. The alpha version is currently 5.0.0-*.  When that changes to v6 there will be explicit commits branching v5 into beta and updating the master to 6.0.0.

The design of snapshot version numeric is to enable clean integration with beta and stable release numberings. So package versioning systems in our downstream distros can handle custom builds and identify builds of v5 alphas from v4 alphas, etc. Many of them have automation performing "a new version is available" notifications to both packagers and administrators based on these numerics.